components/monitoring/logging/base/configure-logging/configure-log-collectors.yaml

```diff
@@ -1,10 +1,9 @@
 ---
-apiVersion: "logging.openshift.io/v1"
-kind: "ClusterLogging"
+apiVersion: "observability.openshift.io/v1"
+kind: "ClusterLogForwarder"
```

Comment:
with this change, we now have 2 ClusterLogForwarder named "instance"

Reply:
This also looks like the reason why kube-linter is failing